src/browser/settings.jsx

```diff
@@ -28,14 +28,21 @@ var SettingsPage = React.createClass({
     } catch (e) {
       config = settings.loadDefault();
     }
+
+    this.setState({
```

Comment:
This causes an warning in the console log.
```
settings.js:982 Warning: setState(...): Can only update a mounted or mounting component. This usually means you called setState() on an unmounted component. This is a no-op. Please check the code for the SettingsPage component.warning @ settings.js:982
```
Removing this code, solves it, not sure if it has any drawbacks/problems/how to do it properly.